src/app/common/transactions/stacks/transaction.utils.ts

```diff
@@ -140,3 +141,7 @@ export function getEstimatedConfirmationTime(
 
   return `~${arrivesIn / 60} min`;
 }
+
+export function isPendingTx(tx: MempoolTransaction): boolean {
+  return tx.tx_status === 'pending';
+}
```

Comment:
A `MempoolTransaction` is always pending, no? 

Doesn't this fn want to allow a wider type (all txs), and infer `MempoolTransaction` from it? 

```suggestion
export function isPendingTx(tx: AllTxTypes): tx is MempoolTransaction {
  return tx.tx_status === 'pending';
}
```

Reply:
This was a function brought over from `@stacks/ui`, I didn't notice the type but can check it.